project/src/main/webapp/components/comment-component.js

```diff
@@ -0,0 +1,52 @@
+import {html, LitElement} from 'https://unpkg.com/@polymer/lit-element/lit-element.js?module';
+
+export class CommentComponent extends LitElement {
+  static get properties() {
+    return {
+      placeholder : {type: String},
+      name : {type: String},
+      time : {type: String}
+    };
+  }
+
+  constructor() {
+    super();
+    this.placeholder = 'Write comment...';
+    this.name ='';
+    this.time = '';
+  }
+
+  // Remove shadow DOM so styles are inherited
+  createRenderRoot() {
+    return this;
+  }
+
+  getServletData() {
+    fetch('/UserHome').then((response) => response.json()).then((documentsData) => { 
+      var date = new Date().toJSON().slice(0,10);
+      var time = new Date().toJSON().slice(11,19)
+      this.time = date + ' ' + time;
+      this.name = documentsData.nickname;
+    });
+  }
+
+  render() {
+    return html` 
+      <div class="comment-div">
+        ${this.getServletData()}
```

Comment:
call this on firstUpdated() if you get latency issues; calling the servlet on each render generally takes some time

Reply:
that is just a placeholder to show what comments look like, we will not actually need to call a servlet